sdcm/utils/common.py

```diff
@@ -1599,28 +1601,29 @@ def get_ami_tags(ami_id, region_name):
             return {}
 
 
-def get_db_tables(session, keyspace_name, node, with_compact_storage=True):
+def get_db_tables(keyspace_name, node, with_compact_storage=None):
     """
-    Return tables from keystore based on their compact storage feature
+    Return tables from keyspace based on their compact storage feature.
     Arguments:
-        session -- DB session
-        ks -- Keypsace name
-        with_compact_storage -- If True, return non compact tables, if False, return compact tables
-
+        keyspace_name -- Keyspace name
+        node -- Node to run CQLSH commands
+        with_compact_storage -- If True, return tables with compact storage; if False, return tables without compact storage; if None, return all tables
     """
     output = []
-    for row in list(session.execute(f"select table_name from system_schema.tables where keyspace_name='{keyspace_name}'")):
-        try:
-            create_table_statement = node.run_cqlsh(f"describe {keyspace_name}.{row.table_name}").stdout.upper()
-        except (UnexpectedExit, Libssh2_UnexpectedExit) as err:
-            # SCT issue https://github.com/scylladb/scylla-cluster-tests/issues/7240
-            # May happen when disrupt_add_remove_dc nemesis run in parallel to the disrupt_add_drop_column
-            LOGGER.error("Failed to describe '%s.%s' table. Maybe the table has been deleted. Error: %s",
-                         keyspace_name, row.table_name, err.result.stderr)
+    try:
+        schema_output = node.run_cqlsh("DESC SCHEMA WITH INTERNALS").stdout
```

Comment:
wondering if  same idea can improve `get_any_ks_cf_list` ?

Reply:
do we face perf issues with it?